x-pack/plugins/maps/public/classes/sources/kibana_regionmap_source/kibana_regionmap_source.ts

```diff
@@ -62,13 +73,13 @@ export class KibanaRegionmapSource extends AbstractVectorSource {
         })
       );
     }
-    return meta;
+    return layerConfig;
   }
 
-  async getGeoJsonWithMeta() {
-    const vectorFileMeta = await this.getVectorFileMeta();
+  async getGeoJsonWithMeta(): Promise<GeoJsonWithMeta> {
+    const vectorFileMeta: LayerConfig = await this.getVectorFileMeta();
```

Comment:
There is no need to define the type of `vectorFileMeta`. TS knows the type since getVectorFileMeta returns `Promise<LayerConfig>`